mava/configs/default/ff_sable_sebulba.yaml

```diff
@@ -0,0 +1,11 @@
+defaults:
+  - logger: logger
+  - arch: sebulba
+  - system: sable/ff_sable
+  - network: ff_retention  # [mlp, continuous_mlp, cnn]
```

Comment:
for network the only option is ff_retention, so if you can remove the comment for `# [mlp, continuous_mlp, cnn]`